app/models/energy/ScheduleFixedInterval.py

```diff
@@ -0,0 +1,63 @@
+"""Schedule Fixed Interval Schema"""
```

Comment:
module names most be all lowercase. This filename should be `schedulefixedinterval.py`. 
if you want to avoid long module names you can create a folder for schedule. copy all the schedule files inside the folder and then name them as `fixedinterval.py` and so on.